packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -1014,40 +1057,59 @@ export class ReadAlongComponent {
    *
    * A sentence element with one or more words
    */
-  Sentence = (props: { words: Node[], attributes: NamedNodeMap }): Element => {
-    if (!this.hasTextTranslations && props.attributes["class"]) {
-      this.hasTextTranslations = props.attributes["class"].value.match("translation") != null;
+  Sentence = (props: { sentenceData: any | null }): Element => {
```

Comment:
Can't we have something more precise than `any` here?

Reply:
yup. good call. I"ll change it.